src/components/TruncatedText.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { TooltipProps } from 'reactstrap';
+import Tooltip from './Tooltip';
+
+
+interface TruncatedTextProps extends TooltipProps {
+  text: string;
+  targetId: string;
+  recordId: string | number;
```

Comment:
I don't think having both targetId and recordId makes sense in this case. We should only really need a prop called id and this logic for making an id that comprises of two variables should be on the user side instead.